.github/workflows/ci.yml

```diff
@@ -27,3 +27,28 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run lint
         run: clj -Alint
+
+  # ref: https://docs.github.com/en/free-pro-team@latest/actions/guides/publishing-docker-images
+  # You can find the image on: https://hub.docker.com/r/xcoo/cruler
+  snapshot-push:
+    name: snapshot push
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

Comment:
https://github.community/t/how-do-i-specify-job-dependency-running-in-another-workflow/16482/2